types/hapi__subtext/index.d.ts

```diff
@@ -0,0 +1,54 @@
+// Type definitions for @hapi/subtext 7.0.3
+// Project: https://github.com/hapijs/subtext#readme
+// Definitions by: Sebastian Malton <https://github.com/nokel81>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
```

Comment:
The documentation for this library is both lacking and out of date: https://hapi.dev/module/subtext/api/?v=7.0.3